package.json

```diff
@@ -68,6 +68,8 @@
     "typescript": "^3.7.3"
   },
   "contributors": [
-    "Angelo Abdoelsamath <angelo.abdoelsamath@mollie.com>"
+    "Angelo Abdoelsamath <angelo.abdoelsamath@mollie.com>",
```

Comment:
Do you need to merge `master` into this branch?

Reply:
It's merged now.